packages/amplify-cli-core/src/utils/open.ts

```diff
@@ -8,14 +8,24 @@ import { isCI } from '..';
  * @param target The thing you want to open. Can be a URL, file, or executable.
  * @param options opn.Options
  */
-export const open = (target: string, options: opn.Options): Promise<ChildProcess | void> => {
+export const open = async (target: string, options: opn.Options): Promise<ChildProcess | void> => {
   if (isCI()) {
     return Promise.resolve();
   }
+  let childProcess: ChildProcess;
   try {
-    return opn(target, options);
+    childProcess = await opn(target, options);
+    childProcess.on('error', (e) => handleOpenError(e, target));
   } catch (e) {
-    console.warn(`Unable to open ${target}: ${(e as Error).message}`);
+    handleOpenError(e, target);
     return Promise.resolve();
   }
+  return Promise.resolve(childProcess);
+};
+
+const handleOpenError = (err: Error, target: string) => {
+  console.error(`Unable to open ${target}: ${err.message}`);
+  if ('code' in err && err['code'] == 'ENOENT') {
```

Comment:
```suggestion
  if (err?.code === 'ENOENT') {
```

Reply:
From the interface, err is just a generic `Error`
`Property 'code' does not exist on type 'Error'`

Reply:
Then my only suggestion would be use `===` over `==`

Reply:
Good point, updated.